sprog.c

```diff
@@ -582,6 +570,12 @@ main(int argc, char *argv[])
 	return EXIT_SUCCESS;
 }
 
+int
+isvowel(int c)
```

Comment:
Please change the return type on this to be `bool`.

Reply:
Done.